std/http/testdata/simple_https_server.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+// This is an example of a https server
+import { serveTLS } from "../server.ts";
+
+const tlsOptions = {
+  hostname: "localhost",
+  port: 4503,
+  certFile: "./http/testdata/tls/localhost.crt",
+  keyFile: "./http/testdata/tls/localhost.key",
+};
+console.log(`Simple HTTPS server listening on ${tlsOptions.hostname}:${tlsOptions.port}`);
```

Comment:
Can you try printing this after serveTLS is called? That might allow removing the delay